apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -56,6 +64,41 @@ ${context}
 `;
 }
 
+async function checkRender(
+  status: 'success' | 'error',
+  errors: string[],
+  courseId: string,
+  userId: string,
+  questionId: string,
+) {
+  // If there was any issue generating the question, we won't yet check rendering.
+  if (status === 'error' || errors.length > 0) return [];
+
+  const question = await selectQuestionById(questionId);
+  const course = await selectCourseById(courseId);
+  const user = await selectUserById(userId);
+  const authnUser = await selectUserById(userId); //user id == authn user id in this case
```

Comment:
Why select it twice? It's going to be exactly the same user.

Reply:
changed in 14c51752ea80ba4c11df5528db78fe5b7b69be62